client/app/containers/HomePage/index.js

```diff
@@ -4,14 +4,14 @@
  * This is the first thing users see of our App, at the '/' route
  */
 
-import React from 'react';
+import React, { PropTypes } from 'react';
 import Helmet from 'react-helmet';
 
 // Components
 import LandingHeader from 'components/LandingHeader';
 import Blog from 'containers/Blog';
 
-const HomePage = () => (
+const HomePage = ({ routeParams }) => (
   <article>
```

Comment:
routeParams is the object passed into the React element by ReactRouter, this is how I'm catching it, then using the pageId property later. Is there a more specific approach, like pulling the pageId key explicitly in the destructuring on this line?